tests/runner/src/main.rs

```diff
@@ -12,31 +12,51 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use std::path::PathBuf;
+
+use clap::Parser;
 use env::Env;
 use sqlness::{ConfigBuilder, Runner};
 
 mod env;
 mod util;
 
+#[derive(Parser, Debug)]
+#[clap(author, version, about, long_about = None)]
+/// SQL Harness for GrepTimeDB
+struct Args {
+    /// Directory of test cases
+    #[clap(short, long)]
+    case_dir: Option<PathBuf>,
+
+    ///Fail this run as soon as one case fails if true
+    #[arg(short, long, action = clap::ArgAction::Count)]
+    fail_fast: u8,
```

Comment:
Prefer to declare this as `bool`. Default to not fail fast, and if this option is given then switch to fail-fast mode